codebuild/bin/build_aws_crt_cpp.sh

```diff
@@ -30,6 +30,8 @@ source codebuild/bin/s2n_setup_env.sh
 BUILD_DIR=$1
 INSTALL_DIR=$2
 
+# Make sure there isn't another source tree hanging around.
+rm -rf /opt/s2n-tls || true
```

Comment:
Can you expand on why we need to do this, why the failure started happening and what happens if we accidentally remove this line.

Maybe also link to your investigation issue.

Reply:
The pre-built docker containers have an s2n-tls clone that was used to build and pre-install the libcrypto's.  We're not cleaning that up at the end of the docker build process (issue coming).  This copy of s2n-tls is ~1 month old, and is being found by the [aws-c-io find_package](https://github.com/awslabs/aws-c-io/blob/main/CMakeLists.txt#L157) portion of CRT's build.  The recent changes appear to have caused a behavior change in cmake.